.changeset/great-eyes-tan.md

```diff
@@ -0,0 +1,42 @@
+---
+"@guardian/cdk": major
+---
+
+The `access_logs.s3.enabled` Application Load Balancer attribute is explicitly set, with a default value of `true`.
+
+Access logging for Application Load Balancers (ALBs) is now enabled by default.
+These logs are helpful during incident triage; therefore, they are now enabled by default.
+
+Previously, access logging was opt-in via the `accessLogging` object property on the `GuEc2App` pattern and its subclasses.
```

Comment:
Minor point, but I think we could make life easier for users here and just name the relevant patterns (as knowing about the subclasses requires knowledge of this library code IIUC):

```suggestion
Previously users of the `GuEc2App`, `GuNodeApp`, `GuPlayApp ` and `GuEc2AppExperimental` patterns could opt-in to this logging via the `accessLogging` property.
```